coderd/testdata/insights/multiple_users_and_workspaces_three_weeks_second_template.json.golden

```diff
@@ -0,0 +1,88 @@
+{
+  "report": {
+    "start_time": "2023-08-01T00:00:00Z",
+    "end_time": "2023-08-22T00:00:00Z",
+    "template_ids": [
+      "00000000-0000-0000-0000-000000000002"
+    ],
+    "active_users": 1,
+    "apps_usage": [
+      {
+        "template_ids": [
+          "00000000-0000-0000-0000-000000000002"
+        ],
+        "type": "builtin",
+        "display_name": "Visual Studio Code",
+        "slug": "vscode",
+        "icon": "/icon/code.svg",
+        "seconds": 3600
+      },
+      {
+        "template_ids": [
+          "00000000-0000-0000-0000-000000000002"
+        ],
+        "type": "builtin",
+        "display_name": "JetBrains",
+        "slug": "jetbrains",
+        "icon": "/icon/intellij.svg",
+        "seconds": 0
+      },
+      {
+        "template_ids": [
+          "00000000-0000-0000-0000-000000000002"
+        ],
+        "type": "builtin",
+        "display_name": "Web Terminal",
+        "slug": "reconnecting-pty",
+        "icon": "/icon/terminal.svg",
+        "seconds": 0
+      },
+      {
+        "template_ids": [
+          "00000000-0000-0000-0000-000000000002"
+        ],
+        "type": "builtin",
+        "display_name": "SSH",
+        "slug": "ssh",
+        "icon": "/icon/terminal.svg",
+        "seconds": 3600
+      },
+      {
+        "template_ids": [
+          "00000000-0000-0000-0000-000000000002"
+        ],
+        "type": "app",
+        "display_name": "app1",
+        "slug": "app1",
+        "icon": "/icon1.png",
+        "seconds": 21600
+      }
+    ],
+    "parameters_usage": []
+  },
+  "interval_reports": [
+    {
+      "start_time": "2023-08-01T00:00:00Z",
+      "end_time": "2023-08-08T00:00:00Z",
```

Comment:
You could consider adding some test data in this interval, just as an extra check that the query is performing as expected.

Reply:
Done 👍